deploy/charts/klicker-uzh-v2/templates/ingress-response-api.yaml

```
@@ -0,0 +1,46 @@
+{{- if .Values.responseApi.ingress.enabled -}}
+{{- $fullName := include "chart.fullname" . -}}
+{{- $svcPort := .Values.responseApi.service.port -}}
+{{- if and .Values.responseApi.ingress.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
+  {{- if not (hasKey .Values.responseApi.ingress.annotations "kubernetes.io/ingress.class") }}
+  {{- $_ := set .Values.responseApi.ingress.annotations "kubernetes.io/ingress.class" .Values.responseApi.ingress.className}}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)